### ISSUE TITLE
Wording...

### ISSUE THREAD

**Issue by mldenne** [jh2642/date_night#3]
Your date - has several different meanings. Maybe say "Who do you want to set a date up with?" or something that refers to a person and not a time or day...

Why so lonely? - Aack! I'm not lonely, just want to set up some time with my hubby. Maybe rephrase to sound like you are going to help someone organize their dating life...or just set up time with a loved one!

**Comment by jh2642** [comment]
Great feedback, I'll work on making it less offensive for you.

**Comment by jh2642** [comment]
This item has been completed. Thanks again for the feedback.
